cordova-plugin-safariviewcontroller.opam

```
@@ -1,5 +1,5 @@
 opam-version: "2.0"
-version: "1.0"
+version: "1.1"
```

```suggestion
version: "2.0"
```

Since it is a breaking change.